### PR TITLE
Update install-and-configure-rbs-with-a-3rd-party-provider.md

### PR DESCRIPTION
**Pull request by tseward**
Formatting and updated links.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/install-and-configure-rbs-with-a-3rd-party-provider.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ You must install RBS client library on all Web servers in the SharePoint farm. T
 
 1. Confirm that the user account performing these steps is a member of the Administrators group on the computer where you are installing the library.
     
-2. On any front-end or application server, for SharePoint Server 2016, download the [Microsoft SQL Server 2014 Feature Pack](https://go.microsoft.com/fwlink/p/?LinkID=733635&amp;clcid=0x409). Run the self-extracting download package to create an installation folder for the X64 RBS.msi file.
+2. On any front-end or application server, for SharePoint Server 2016, download the [Microsoft SQL Server 2014 Feature Pack](https://go.microsoft.com/fwlink/p/?LinkID=733635). Run the self-extracting download package to create an installation folder for the X64 RBS.msi file.
     
-    For SharePoint 2013, [download the RBS_amd64.msi file](https://go.microsoft.com/fwlink/p/?LinkId=271938).
+    For SharePoint 2013, [download the RBS.msi file](https://www.microsoft.com/en-us/download/details.aspx?id=56041).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/install-and-configure-rbs-with-a-3rd-party-provider.md:60]:
```suggestion
    For SharePoint 2013, [download the RBS.msi file](https://www.microsoft.com/download/details.aspx?id=56041).
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointServer/administration/install-and-configure-rbs-with-a-3rd-party-provider.md:60]: Please remove the locale.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**